client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -639,7 +639,7 @@ export const getMonitorDowntimeText = ( downtime: number | undefined ) => {
 
 	return translate( 'Downtime for %(time)s', {
 		args: {
-			time,
+			time: time.trim(),
```

Comment:
Since we add text like "1d" or "2h" only, an extra space gets added at the end which looks like "1d " or "2h " which is not necessary.